src/Confluent.Kafka/Producer.cs

```diff
@@ -32,7 +32,7 @@ namespace Confluent.Kafka
     /// <summary>
     ///     Implements a high-level Apache Kafka producer (without serialization).
     /// </summary>
-    public class Producer : IDisposable
+    public class Producer : IDisposable, IAdmin
```

Comment:
I don't think we should expose admin operations on `Producer`, only on `AdminClient`.

Reply:
Oh, explicit implementation. Probably better. Still, I was thinking of having a `Handle` property and allowing AdminClient to be constructed from this. Have not completely thought through all the details though.

Reply:
As I say, it's a discussion ;)
What I would like is to be able to use admin method from existing handler - an IAdminClient AdminHelper.GetAdmin(Producer producer) will work fine for this -and this is where explicit implementation is nice, you just have to return producer as IAdmin.
Here, we can't write producer.DeleteTopics - producer must be cast to IAdmin first

Other reason is that we will have to list what is admin api and what is not (is ListGroup/QueryWatermarkOffsets/etc an admin API? should we remove it / depreciate it from existing Consumer/Producer?)

Reply:
Yep - we make a call on what belongs where typical need and remove / depreciate from Consumer/Producer as appropriate. All methods that may be affected are currently marked UNSTABLE, so it's no problem.